### PR TITLE
updated nF core to 1.10.4-preview.11

### PR DESCRIPTION
**Pull request by rschiefer**

### CODE REVIEW

File: UnitsNet.NanoFramework/GeneratedCode/Acceleration/Acceleration.nfproj [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProjectTypeGuids>{11A8DD76-328B-46DF-9F39-F559912D0360};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
-    <ProjectGuid>{3985716e-fa45-4c63-92a7-d656cc46d421}</ProjectGuid>
+    <ProjectGuid>{9f2a32c7-6fcd-4987-ab43-c0b18d616762}</ProjectGuid>
```
[the remark this answers is not in the document]
Comment (angularsen) [reply UnitsNet.NanoFramework/GeneratedCode/Acceleration/Acceleration.nfproj:11]: I pushed a fix for deterministic project/solution guids. Please re-generate.

**Review by angularsen — CHANGES_REQUESTED** [review]

**Comment by rschiefer** [comment]
Think I've got it all in and correct.  Apologies if not, still new to UnitsNet and nanoFramework so please bear with me.

**Comment by angularsen** [comment]
Found another issue with the codegen, running codegen locally for me found `1.10.4-preview.4` as latest version instead of preview.11. Fixed in master and merged in here.

**Comment by angularsen** [comment]
No need to apologize! This is great work. Most of the problems you ran into were due to the codegen having some minor bugs and inconsistencies that are now fixed as a result of your PR. nanoFramework is still very new to us.

Nuget is hopefully on the way out, within 30 minutes or so. 🎉
https://ci.appveyor.com/project/angularsen/unitsnet/history

[Release UnitsNet/4.92.0 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F4.92.0)

**Comment by rschiefer** [comment]
Awesome, thanks SO MUCH!

**Pull request merged**